pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -94,6 +97,7 @@ func NewReconciler(
 	nodeName types.NodeName,
 	desiredStateOfWorld cache.DesiredStateOfWorld,
 	actualStateOfWorld cache.ActualStateOfWorld,
+	populatorHasAddedPods func() bool,
```

Comment:
Ah, I missed that reconciler is reading the value. In that case yeah, we perhaps do need the lock.